src/pages/docs/ai-transport/index.mdx

```diff
@@ -4,3 +4,56 @@ meta_description: "Learn more about Ably's AI Transport and the features that en
 redirect_from:
   - /docs/products/ai-transport
 ---
+
+Ably AI Transport is a solution for building stateful, steerable, multi-device AI experiences into new or existing applications. You can use AI Transport as the transport layer with any LLM or agent framework, without rebuilding your existing stack or being locked to a particular vendor.
+
+## Key features
+
+AI Transport builds on [Ably Pub/Sub](/docs/basics) to enable reliable interactive AI experiences at scale and provides the following key features:
+
+// TODO: Check this lines up with planned doc sections
+* [Reliable, resumable token streams](#reliable-tokens)
+* [Session management](#session)
+* [Complex message patterns](#message)
+* [Enterprise controls](#enterprise)
+
```

Comment:
Should these not be "Advanced messaging" and "User input" ? (Maybe user input isn't a helpful title).
But they're the sections defined in the AIT Docs IA Miro.